packages/server/test/unit/plugins/index_spec.js

```diff
@@ -113,7 +113,8 @@ describe('lib/plugins/index', () => {
       })
     })
 
-    it('sends \'load\' event with config via ipc', () => {
+    it('sends \'load\' event with config via ipc once it receives \'ready\'', () => {
+      ipc.on.withArgs('ready').yields([])
```

Comment:
This test seems to pass without the changes in this PR. Can you add a test that demonstrates the failing case fixed by this PR (asynchronous plugin registration)?

Reply:
You're right. I'll try to fix that in the next couple of days.

Reply:
I've updated the test. Ready to be reviewed again :)